wp-content/themes/eighshift-task/src/blocks/components/heading/components/heading-editor.js

```diff
@@ -7,7 +7,6 @@ export const HeadingEditor = (props) => {
     blockClass,
     content,
     onChangeContent,
-    level,
```

Comment:
why did you remove level?

Reply:
My bad. Probably because linter was throwing warnings.